sci-libs/openblas/openblas-0.2.14.ebuild

```diff
@@ -108,12 +108,12 @@ src_compile() {
 		# cflags already defined twice
 		unset CFLAGS || die
 		emake clean && emake libs shared ${openblas_flags}
-		mkdir -p libs && mv libopenblas* libs/ || die
+		mkdir -p libs && mv libopenblas* libs/ ||??die
 		# avoid pic when compiling static libraries, so re-compiling
 		if use static-libs; then
 			emake clean
 			emake libs ${openblas_flags} NO_SHARED=1 NEED_PIC=
-			mv libopenblas* libs/ || die
+			mv libopenblas* libs/ ||??die
 		fi
```

Comment:
I use a mix of kwrite and nano. My money is on kwrite doing something weird, but that implies that there is something special about that space - or its surroundings - to start with.